enforcer-rules/src/it/bom-project-error/verify.groovy

```diff
@@ -3,8 +3,10 @@ def buildLog = new File(basedir, "build.log").text.replaceAll("\\r\\n", "\n")
 assert buildLog.contains('''\
 [ERROR] Linkage Checker rule found 1 error. Linkage error report:
 (com.google.guava:guava:20.0) com.google.common.base.Verify's method verify(boolean arg1, String arg2, Object arg3) is not found;
-  referenced by 1 class file
+  referenced by 3 class files
     io.grpc.internal.ServiceConfigInterceptor (io.grpc:grpc-core:1.17.1)
+    io.grpc.internal.JndiResourceResolverFactory (io.grpc:grpc-core:1.17.1)
+    io.grpc.internal.DnsNameResolver (io.grpc:grpc-core:1.17.1)
```

Comment:
This is the class that raises NoSuchMethodError in the example project https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/example-problems/no-such-method-error-signature-mismatch